### DISCUSSION TITLE
Flutter & Dart - Développement piloté par les tests

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#178] [Announcements]
# Flutter & Dart - Développement piloté par les tests

Apprenez à écrire du code propre et maintenable en utilisant le TDD pour vos applications Flutter & Dart.

https://zerofiltre.tech/articles/253

**Comment by imphilippesimo** [comment]
## c/ Code genPwd() final

*Avant cette section, ajoute encore au moins deux processus de Red, green ,refactor, tu n'en as montré qu'un*  
*En effet, un exemple n'est pas suffisant, il faut que le lecteur puisse bien comprendre comment faire du red green refactor avec des sénario supplémentaires*
